raiden/api/python.py

```diff
@@ -1696,13 +1696,21 @@ def create_light_client_payment(
         amount: typing.TokenAmount,
         secrethash: typing.SecretHash
     ) -> HubResponseMessage:
+
         channel_state = views.get_channelstate_for(
             views.state_from_raiden(self.raiden),
             registry_address,
             token_address,
             creator_address,
             partner_address,
         )
+        if not channel_state:
```

Comment:
Could you please explain why is this necessary?

Reply:
In the first "try", he'll check if he has a channel with the partner, that if is necessary for mediated transfers. It will return the first channel that the node has, and try mediate the payment.

Reply:
I think thats related with the modification of that method. But, i dont see a need for a "retry" on a method invocation. The method should be invoked with the right parameters and it should handle the situation.

Reply:
Please re-review this, i created a function to get the first channel_state in case that channel_state is none